events/schema.py

```diff
@@ -496,11 +561,33 @@ def mutate_and_get_payload(cls, root, info, **kwargs):
 
 class Query:
     events = DjangoFilterConnectionField(EventNode)
+    events_and_event_groups = graphene.ConnectionField(
+        EventOrEventGroupConnection, project_id=graphene.ID()
+    )
     occurrences = DjangoFilterConnectionField(OccurrenceNode)
 
     event = relay.Node.Field(EventNode)
+    event_group = relay.Node.Field(EventGroupNode)
     occurrence = relay.Node.Field(OccurrenceNode)
 
+    def resolve_events_and_event_groups(self, info, **kwargs):
+        event_qs = Event.objects.filter(event_group=None)
+        event_group_qs = EventGroup.objects.all()
+
+        if "project_id" in kwargs:
+            project_id = get_node_id_from_global_id(kwargs["project_id"], "ProjectNode")
```

Comment:
Maybe it's not really a big issue, but I think you can call this query explicitly with `project_id=null` which probably causes an exception from `get_node_id_from_global_id(None)`

Reply:
Good point, fixed.